osidb/migrations/0123_history_audit.py

```diff
@@ -0,0 +1,869 @@
+         AND   acl_write::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]);
+    --policy for entity select
+    DROP policy if exists acl_policy_affectaudit_select on osidb_affectaudit;
+    create policy acl_policy_affectaudit_select
+    on osidb_affectaudit
+    for select
+    USING (acl_read::uuid[] && string_to_array(current_setting('osidb.acl'),',')::uuid[]);
+    --policy for entity update
+    DROP policy if exists acl_policy_affectaudit_update on osidb_affectaudit;
+    create policy acl_policy_affectaudit_update
+    on osidb_affectaudit
+    for update
+    USING (acl_write::uuid[] && string_to_array(current_setting('osidb.acl'),',')::uuid[])
+    WITH CHECK (acl_read::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]
+         AND   acl_write::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]);
+    --policy for entity delete
+    DROP policy if exists acl_policy_affectaudit_delete on osidb_affectaudit;
+    create policy acl_policy_affectaudit_delete
+    on osidb_affectaudit
+    for delete
+    USING (acl_write::uuid[] && string_to_array(current_setting('osidb.acl'),',')::uuid[]);
+
+    --enable row based security for flawmetaaudit entity table
+    ALTER TABLE osidb_flawmetaaudit ENABLE ROW LEVEL SECURITY;
+    ALTER TABLE osidb_flawmetaaudit FORCE ROW LEVEL SECURITY;
+    --following policies define fine grained read/write control on osidb_flawmetaaudit entity
+    --policy for entity insert (eg. create)
+    DROP policy if exists acl_policy_meta_create on osidb_flawmetaaudit;
+    create policy acl_policy_meta_create
+    on osidb_flawmetaaudit
+    for INSERT
+    WITH CHECK (acl_read::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]
+         AND   acl_write::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]);
+    --policy for entity select
+    DROP policy if exists acl_policy_meta_select on osidb_flawmetaaudit;
+    create policy acl_policy_meta_select
+    on osidb_flawmetaaudit
+    for select
+    USING (acl_read::uuid[] && string_to_array(current_setting('osidb.acl'),',')::uuid[]);
+    --policy for entity update
+    DROP policy if exists acl_policy_meta_update on osidb_flawmetaaudit;
+    create policy acl_policy_meta_update
+    on osidb_flawmetaaudit
+    for update
+    USING (acl_write::uuid[] && string_to_array(current_setting('osidb.acl'),',')::uuid[])
+    WITH CHECK (acl_read::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]
+         AND   acl_write::uuid[] && string_to_array(current_setting('osidb.acl'), ',')::uuid[]);
+    --policy for entity delete
+    DROP policy if exists acl_policy_meta_delete on osidb_flawmetaaudit;
+    create policy acl_policy_meta_delete
+    on osidb_flawmetaaudit
+    for delete
+    USING (acl_write::uuid[] && string_to_array(current_setting('osidb.acl'),',')::uuid[]);
+    """,
+        ),
+    ]
```

Comment:
I think I will trust that this migration does all it should :smile: this is above my knowledge.